bdb/rep.c

```diff
@@ -2359,6 +2359,48 @@ int verify_master_leases(bdb_state_type *bdb_state, const char *func,
 int gbl_catchup_window_trace = 0;
 extern int gbl_set_seqnum_trace;
 
+static inline int copy_seqnum(bdb_state_type *bdb_state, seqnum_type *seqnum,
+                              int node_ix)
+{
+    int trace = bdb_state->attr->wait_for_seqnum_trace, now;
+    int last_generation = bdb_state->seqnum_info->seqnums[node_ix].generation;
```

Comment:
Pass bdb_state->seqnum_info->seqnums[node_ix] and you only need 2 parameters to this function.
In fact you can do the memcopy here at the end of the func before returning.